### ISSUE TITLE
"tyt" bug ?

### ISSUE THREAD

**Issue by Judykator** [ManiacDC/TypingAid#80]
I use TypingAid 2.20 With the newer version had the problems about which I have written.
Now I think I found a bug that difficult for me to use the program.

The program does not save (ctrl + shift + c) and does not learn phrases that begin the phrase "tyt"
for example:
tytułem 
tytan
tytuł
tytaniczny 
tytus
It did not help even save those words into a file WordlistLearned.txt

Program window does not appear when I type three letters "tyt" 

This is so at least in the programs: Word, Evernote, Firefox

In other words, I have not noticed problems with remembering words or phrases (ctrl + shift + c)

**Comment by ManiacDC** [comment]
Hmm, works fine in 2.22 for me. Are you using a standard english keyboard layout?

**Comment by Judykator** [comment]
YES. Qwerty.

**Comment by ManiacDC** [comment]
1. Can you try it with 2.22 and see if it happens?
2. If so, if you switch your keyboard to US qwerty, do you have the same issue? Also, can you attach your preferences and wordlist please? I'll take a look.

**Comment by Judykator** [comment]
This is what I wrote has on your work computer (Windows 7). Now I'm at home on my computer (Windows 7 too) everything works.

Synchronizes both computers (folder TypingAid) so files should be identical.

[Preferences.zip](https://github.com/ManiacDC/TypingAid/files/432441/Preferences.zip)

Wordlist is too private.

**Comment by Judykator** [comment]
I am now back at work ...
v. 2.20 phrase "tyt" did not work
v. 2.22 phrase "tyt" ... works

**Comment by ManiacDC** [comment]
Administrator vs non-administrator should only matter when it comes to actually picking the word... not popping up the list of choices.

**Comment by ManiacDC** [comment]
Do you use the same keyboard layout at home and at work?

**Comment by Judykator** [comment]
Yes. 
At home Windows 7 Windows 7 Home Premium , at work Windows 7 Professional

But now after the change in Ditto I can use at work v.2.22. Then "tyt" bug do not occur. However, this will test after three weeks, because tomorrow I go to vacation.

**Comment by ManiacDC** [comment]
OK, let me know if everything is all set or if you're still having issues.

**Comment by Judykator** [comment]
With version 2.22 is OK.

**Comment by ManiacDC** [comment]
Great thanks, I'll close this issue.
